cuvslam-skills/cuvslam-troubleshoot/scripts/stereo_image_throttle.py

```
@@ -0,0 +1,146 @@
+ROS 2 parameters
+----------------
+target_hz   float  Target publish rate (default 30.0).
+                   Recommended: ≥ 60.0 for VIO; any value for VO-only.
+input_left  str    Input left-image topic  (default /camera/infra1/image_rect_raw)
+input_right str    Input right-image topic (default /camera/infra2/image_rect_raw)
+output_left  str   Output left-image topic  (default <input_left>_throttled)
+output_right str   Output right-image topic (default <input_right>_throttled)
+
+image_jitter_threshold_ms guideline
+------------------------------------
+Set the SLAM node's image_jitter_threshold_ms to at least 3 × frame_period_ms
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)